### DISCUSSION TITLE
Amazing speed boost in Alpha 0.81.0-2196

### DISCUSSION THREAD

**Discussion by almeath** [dosbox-staging/dosbox-staging#3251] [General]
I recently switched to Windows from MacOS for all my gaming and emulation needs, and I noticed right away that all my installs of DOSBox Staging, X and SVN were performing quite a lot slower than on the Mac. I have spent weeks trying to figure it out, but without success. Many of my Windows 9x DOSBox installs (especially with 3DFX emulation) were noticeably lagging, often with stuttering sound, even when on "max" CPU speed.

Then, I installed 0.81.0 alpha-2196 the other day, and WOW .. my benchmark results have literally doubled, and all the 9x installs are working even faster than they were on the Mac. I went back to test my other DOSBox variants (including the current release version of Staging) and they are still lagging behind.

Whatever has been done to the latest alpha to boost the dynamic core performance in Windows, please keep the changes! Thanks for all the hard work on this. If this keeps up then Staging is going to be my default DOSBox variant going forward. :-)

**Comment by johnnovak** [comment]
That's quite some endorsement @almeath . Glad to hear Staging is working so well for you! 🚀

**Comment by Grounded0** [comment]
This explains why this is our best performing release to date:

https://dosbox-staging.github.io/releases/release-notes/0.81.0-rc/

**Reply by almeath** [reply]
That is some great work. I particularly like the merging of "vgaonly" benefits into the standard VGA support.
